### ISSUE TITLE
[Feature Request] Social Popup interaction

### ISSUE THREAD

**Issue by joebetsill** [MilleXIV/XIV_Databar#6]
Interaction with the social popups, e.g., left click to whisper a guildie, shift+left click to invite a friend, etc.

**Comment by Mikeprod** [comment]
I don't think it is part of the purpose of the addon, but I am just a collaborator at this point. I think we'll see when hearing back from MilleXIV

**Comment by joebetsill** [comment]
My thinking was (in case it helps) that the specialization popup is actionable, so it wasn't too far off the mark to have clicking on a guild member's name be actionable as well.

**Comment by Mikeprod** [comment]
Ah ... I thought of a chat functionality ... As you explain I think it is possible

**Comment by Mikeprod** [comment]
I will wait for the code refactoring to implement your suggestion.
The frame type has to change in order to click on it.

**Comment by joebetsill** [comment]
Sweet, thanks!

**Comment by skmzarn** [comment]
I would love to have this as well. The SLDT addon has a similar function. A dropdown-menu, and you click on the name in the list (guild- or friendlist) to start a whisper with them, and if you alt+left click, you will invite them. That's it - it worked wonder, and I miss this function.

**Comment by Mikeprod** [comment]
That's the next feature I will develop

**Comment by Mikeprod** [comment]
Ok I have done it, so far left click whispers in game or BNet. What do you want more ? Invite and what else ?

**Comment by skmzarn** [comment]
Alt+left click to invite the mouseovered person to a party, other than that, I don't know. That's a really good start. Do you want to share it? I want to test it right away. :)

**Comment by Mikeprod** [comment]
I'll release it tomorrow on my improvement branch

**Comment by skmzarn** [comment]
Very nice, thank you!

**Comment by Mikeprod** [comment]
I eventually pushed my work. I need testers :D I cannot test invitations via Shift+Click with the social popup nor with the guild one. And I need the guild tooltip to be tested too, no guild functions available on starter editions XD Feedbacks appreciated

**Comment by skmzarn** [comment]
I just tested this, and it seems to work fine so far. One thing though, before you could interact with the micromenues and tooltips IN COMBAT, now you can't anymore. Is this intended? I did much prefer the old way. I will continue testing this and give feedback.

Edit: I commented out all of the if InCombatLockdown() temporarily, then it seems to work like I want it to.

Edit 2: I noticed that if a person on your RealID is ingame, and you leftclick that name to whisper, the addon prefers to whisper the ingame (name-realm), when it instead should be the actual realid-name.

**Comment by Mikeprod** [comment]
Thank you for your feedback. As far as I can remember tooltips and the bar function has been limited when in combat. But I can add an option for that in the Social and guild tooltip case. 

Concerning your edit 2 that's what I did, I will do then left click whisp BNet, right click whisp IG, shift left-click invite. 

Any feedback concerning the guild one I could not test ?

**Comment by skmzarn** [comment]
That would be great to add an option for the combat. Aha, I see, that seems understandable and perfectly fine. Leftclick for realID whisper and rightclick for IG whisp, and shift-leftclick for invite.

I changed it to alt+leftclick for invite though, prefer it more. Maybe you could add an option for what modifier you want to use for this?

Inviting from the guild-tooltip works, same with the whisper. Right now I see no more issues than the ones I've mentioned. How about the GMOTD in the guild-tooltip?

**Comment by Mikeprod** [comment]
I added all you asked for. I do not know how the GMOTD will appear though.

**Comment by skmzarn** [comment]
I just tested it. It looks like this: 
http://i.imgur.com/jvyKmL3.jpg

You did a great job with the options and the informative helpers in the tooltips. I would suggest you try to cut the GMOTD into smaller parts, and have only the GMOTD in a smaller fontsize, just to make the whole tooltip smaller. Maybe if you want headers for the "name/level", "note/officernote", "location". I don't know if it's necessary, or if it will make things look better, but it's an idea. What do you think? I love it right now. :)

Edit: Here's how my old databroker addon looked like, to give you some ideas, if you like them: http://i.imgur.com/wlVLFZ3.jpg

**Comment by Mikeprod** [comment]
I'll see what I can do about that GMOTD, not an easy task to divide a string into pieces to fit in columns... Or maybe reduce the font indeed.
The pink color was fixed with the last commit ;) and I know I messed up other colors too. 

Thank you for the tooltip example. Such thing can be done. But I'll do some localisation about the features I developed first.

**Comment by skmzarn** [comment]
Sure, it's no problem and no rush. You've done more than enough already. :)

**Comment by Mikeprod** [comment]
I am almost done with the kind of display you wanted @skmzarn :)

**Comment by skmzarn** [comment]
Wow, that is so cool, thank you!

**Comment by Mikeprod** [comment]
The whole bar is not finished and the options are not all there, but you can check how the tooltip feels by downloading this version  : https://github.com/Mikeprod/XIV_Databar/tree/Rework
